### PR TITLE
Rebrand CAM as MTC

### PR DESCRIPTION
**Pull request by apinnick**
Change "Cluster Application Migration" tool (CAM) to "Migration Toolkit for Containers". This does not affect CPMA.

CP to 4.2, 4.3, 4.4, 4.5.

### CODE REVIEW

File: migration/migrating_3_4/about-migration.adoc [hunk 1]
```diff
@@ -14,5 +14,5 @@ Learn about the differences between {product-title} versions 3 and 4. Prior to t
 
 xref:../../migration/migrating_3_4/migrating-application-workloads-3-to-4.adoc#migrating-application-workloads-3-to-4[Performing your migration]::
 Learn about and use the tools to perform your migration:
-* Cluster Application Migration (CAM) tool to migrate your application workloads
-* Control Plane Migration Assistant (CPMA) to migrate your control plane
+* {mtc-first} to migrate your application workloads
```
Comment (RichardHoch) [inline migration/migrating_3_4/about-migration.adoc:17]: {mtc-first} = Cluster Application Migration (CAM) tool [OpenShift 3 to 4]

File: migration/migrating_3_4/migrating-application-workloads-3-to-4.adoc [hunk 2]
```diff
@@ -6,9 +6,9 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can migrate application workloads from {product-title} 3.7 (and later) to {product-title} {product-version} with the Cluster Application Migration (CAM) tool. The CAM tool enables you to control the migration and to minimize application downtime.
+You can migrate application workloads from {product-title} 3.7 (and later) to {product-title} {product-version} with the {mtc-first}. The {mtc} enables you to control the migration and to minimize application downtime.
```
Comment (RichardHoch) [inline migration/migrating_3_4/migrating-application-workloads-3-to-4.adoc:9]: {mtc-first} = Cluster Application Migration (CAM) tool
{mtc} = CAM tool [OpenShift 3 to 4]

File: migration/migrating_3_4/migrating-application-workloads-3-to-4.adoc [hunk 3]
```diff
@@ -6,9 +6,9 @@ include::modules/common-attributes.adoc[]
...
 
-The CAM tool's web console and API, based on Kubernetes Custom Resources, enable you to migrate stateful application workloads at the granularity of a namespace.
+The {mtc}'s web console and API, based on Kubernetes Custom Resources, enable you to migrate stateful application workloads at the granularity of a namespace.
```
Comment (RichardHoch) [inline migration/migrating_3_4/migrating-application-workloads-3-to-4.adoc:11]: {mtc} = CAM tool [OpenShift 3 to 4]

File: migration/migrating_3_4/migrating-application-workloads-3-to-4.adoc [hunk 4]
```diff
@@ -18,6 +18,6 @@ Before you begin your migration, be sure to review the information on xref:../..
 ====
 
 include::modules/migration-prerequisites.adoc[leveloffset=+1]
-include::modules/migration-understanding-cam.adoc[leveloffset=+1]
+include::modules/migration-understanding-mtc.adoc[leveloffset=+1]
```
Comment (RichardHoch) [inline migration/migrating_3_4/migrating-application-workloads-3-to-4.adoc:21]: mtc = the Cluster Application Migration (CAM) tool [OpenShift 3 to 4]

File: modules/migration-understanding-mtc.adoc [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+The {mtc-first} enables you to migrate Kubernetes resources, persistent volume data, and internal container images from an {product-title} source cluster to an {product-title} {product-version} target cluster, using the {mtc-short} web console or the Kubernetes API.
+
+Migrating an application with the {mtc-short} web console involves the following steps:
+
+. Install the {mtc-operator} on all clusters.
++
+[NOTE]
+====
+The {mtc-operator} installs the {mtc-short} web console and the Migration controller on the target cluster by default. You can configure the Migration controller to install the {mtc-short} on another cluster.
```
Comment (RichardHoch) [inline modules/migration-understanding-mtc.adoc:17]: This note appears in the preview as 

The Cluster Application Migration Operator installs the CAM tool (CAM web console and Migration controller) on the target cluster by default. You can configure the Migration controller to install the CAM tool on another cluster. The cluster on which the CAM tool is installed is referred to as the CAM tool cluster.

Not only are the variables returned oddly, but there is an extra sentence.

File: modules/migration-understanding-mtc.adoc [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+====
+
+. Configure the replication repository, an intermediate object storage that the {mtc-short} uses to migrate data.
+. Add the source cluster to the {mtc-short} web console.
+. Add the replication repository to the {mtc-short} web console.
+. Create a migration plan, with one of the following data migration options:
+
+* *Copy*: The {mtc-short} copies the data from the source cluster to the replication repository, and from the replication repository to the target cluster.
```
Comment (RichardHoch) [inline modules/migration-understanding-mtc.adoc:25]: {mtc-short} = CAM tool

File: modules/migration-understanding-mtc.adoc [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
++
+image::migration-PV-copy.png[]
+
+* *Move*: The {mtc-short} unmounts a remote volume (for example, NFS) from the source cluster, creates a PV resource on the target cluster pointing to the remote volume, and then mounts the remote volume on the target cluster. Applications running on the target cluster use the same remote volume that the source cluster was using. The remote volume must be accessible to the source and target clusters.
```
Comment (RichardHoch) [inline modules/migration-understanding-mtc.adoc:29]: {mtc-short} = CAM tool

File: modules/migration-understanding-data-copy-methods.adoc [hunk 8]
```diff
@@ -6,12 +6,12 @@
 [id='migration-understanding-data-copy-methods_{context}']
 = Understanding the data copy methods for migration
 
-The CAM tool supports the file system and snapshot data copy methods for migrating data from the source cluster to the target cluster. You can select a method that is suited for your environment and is supported by your storage provider.
+The {mtc-first} supports the file system and snapshot data copy methods for migrating data from the source cluster to the target cluster. You can select a method that is suited for your environment and is supported by your storage provider.
```
Comment (RichardHoch) [inline modules/migration-understanding-data-copy-methods.adoc:9]: {mtc-first} = CAM tool

File: modules/migration-understanding-data-copy-methods.adoc [hunk 9]
```diff
@@ -6,12 +6,12 @@
...
 
 [id='file-system-copy-method_{context}']
 == File system copy method
 
-The CAM tool copies data files from the source cluster to the replication repository, and from there to the target cluster.
+The {mtc-short} copies data files from the source cluster to the replication repository, and from there to the target cluster.
```
Comment (RichardHoch) [inline modules/migration-understanding-data-copy-methods.adoc:14]: {mtc-short} = CAM tool

File: modules/migration-understanding-data-copy-methods.adoc [hunk 10]
```diff
@@ -25,7 +25,7 @@ a|* Slower than the snapshot copy method
 [id='snapshot-copy-method_{context}']
 == Snapshot copy method
 
-The CAM tool copies a snapshot of the source cluster's data to a cloud provider's object storage, configured as a replication repository. The data is restored on the target cluster.
+The {mtc-short} copies a snapshot of the source cluster's data to a cloud provider's object storage, configured as a replication repository. The data is restored on the target cluster.
```
Comment (RichardHoch) [inline modules/migration-understanding-data-copy-methods.adoc:28]: {mtc-short} = CAM tool

File: modules/migration-installing-mtc-operator-ocp-3.adoc [hunk 11]
```diff
@@ -29,13 +29,13 @@ If your system is configured for rootless Podman containers, `sudo` is not requi
 . Download the `operator.yml` file:
 +
 ----
-$ sudo podman cp $(sudo podman create registry.redhat.io/rhcam-1-1/openshift-migration-rhel7-operator:v1.1):/operator.yml ./
+$ sudo podman cp $(sudo podman create registry.redhat.io/rhmtc-1-1/openshift-migration-rhel7-operator:v1.1):/operator.yml ./
```
Comment (RichardHoch) [inline modules/migration-installing-mtc-operator-ocp-3.adoc:32]: rhmtc=rhcam in preview.

File: modules/migration-installing-mtc-operator-ocp-3.adoc [hunk 12]
```diff
@@ -29,13 +29,13 @@ If your system is configured for rootless Podman containers, `sudo` is not requi
...
 ----
 
 . Download the `controller-3.yml` file:
 +
 ----
-$ sudo podman cp $(sudo podman create registry.redhat.io/rhcam-1-1/openshift-migration-rhel7-operator:v1.1 ):/controller-3.yml ./
+$ sudo podman cp $(sudo podman create registry.redhat.io/rhmtc-1-1/openshift-migration-rhel7-operator:v1.1 ):/controller-3.yml ./
```
Comment (RichardHoch) [inline modules/migration-installing-mtc-operator-ocp-3.adoc:38]: rhmtc=rhcam in preview.

File: modules/migration-installing-mtc-operator-ocp-4.adoc [hunk 13]
```diff
@@ -1,26 +1,26 @@
...
 
 ifdef::targetcluster-3-4,targetcluster-4_2-4_x,targetcluster-4_1-4_x[]
-= Installing the Cluster Application Migration Operator on an {product-title} {product-version} target cluster
+= Installing the {mtc-operator} on an {product-title} {product-version} target cluster
 
-You can install the Cluster Application Migration Operator on an {product-title} {product-version} target cluster with OLM.
+You can install the {mtc-operator} on an {product-title} {product-version} target cluster with OLM.
 
-The Cluster Application Migration Operator installs the CAM tool on the target cluster by default. If you want to install the CAM tool on a different cluster, you must update the Migration controller configuration so that the Cluster Application Migration Operator does not install the CAM tool on this cluster.
+The {mtc-operator} installs the {mtc-short} on the target cluster by default. If you want to install the {mtc-short} on a different cluster, you must update the Migration controller configuration so that the {mtc-operator} does not install the {mtc-short} on this cluster.
```
Comment (RichardHoch) [inline modules/migration-installing-mtc-operator-ocp-4.adoc:23]: {mtc-short} = CAM tool

File: modules/migration-launching-mtc.adoc [hunk 14]
```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_3_4/migrating-applications-with-mtc.adoc
+// migration/migrating_4_1_4/migrating-applications-with-mtc.adoc
+// migration/migrating_4_2_4/migrating-applications-with-mtc.adoc
+[id="migration-launching-mtc_{context}"]
+= Launching the {mtc-short} web console
+
+You can launch the {mtc-short} web console that is installed on the target cluster.
```
Comment (RichardHoch) [inline modules/migration-launching-mtc.adoc:9]: Preview has: "You can launch the CAM web console in a browser."

File: migration/migrating_3_4/migrating-application-workloads-3-to-4.adoc (hunk 3 above)
Comment (RichardHoch) [inline migration/migrating_3_4/migrating-application-workloads-3-to-4.adoc:11]: appears as {mtc}

File: modules/migration-adding-cluster-to-mtc.adoc [hunk 15]
```diff
@@ -1,12 +1,12 @@
...
+// migration/migrating_3_4/migrating-applications-with-mtc.adoc
+// migration/migrating_4_1_4/migrating-applications-with-mtc.adoc
+// migration/migrating_4_2_4/migrating-applications-with-mtc.adoc
+[id='migration-adding-cluster-to-mtc_{context}']
+= Adding a cluster to the {mtc-short} web console
 
-You can add a source cluster to the CAM web console.
+You can add a source cluster to the {mtc-short} web console.
 
 .Prerequisites for Azure
```
Comment (RichardHoch) [inline modules/migration-adding-cluster-to-mtc.adoc:12]: In the preview, the Prerequisites for Azure are a paragraph with sentences separated by asterisks, not a bulleted list,

Comment (apinnick) [reply modules/migration-adding-cluster-to-mtc.adoc:12]: I see a bullet list

File: migration/migrating_3_4/migrating-application-workloads-3-to-4.adoc (hunk 3 above)
Comment (apinnick) [reply migration/migrating_3_4/migrating-application-workloads-3-to-4.adoc]: Fixed.

If you see an error like this, check the attributes

**Pull request closed without merging**